libraries/pod/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! Crate containing `Pod` types and `bytemuck` utils used in SPL
+
+pub mod non_zero_pubkeys;
```

Comment:
nit: to me, the optionality is the most important part, and not that they're non-zero, so I'd go with `optional_keys` or just `keys`